controllers/configmap_controller.go

```diff
@@ -121,19 +122,36 @@ func (r *ConfigMapReconciler) Reconcile(ctx context.Context,
 	}
 
 	// Fetch the ConfigMap. The predicate will have filtered out any ConfigMaps that we should not reconcile
-	// so it is safe to assume that all ConfigMaps being reconciled describe hosts that need to be present in the
-	// cluster. This also handles the case when the reconciliation is kicked off by the InstanceConfigMap being deleted.
-	// In the deletion case, an empty InstanceConfigMap will be reconciled now resulting in all existing BYOH nodes
-	// being deleted.
+	// so it is safe to assume that all ConfigMaps being reconciled are one of:
+	// 1. windows-instances, describing hosts that need to be present in the cluster.
+	// 2. windows-services, describing expected configuration of WMCO-managed services on all Windows instances
 	configMap := &core.ConfigMap{}
 	if err := r.client.Get(ctx, req.NamespacedName, configMap); err != nil {
 		if !k8sapierrors.IsNotFound(err) {
 			// Error reading the object - requeue the request.
 			return ctrl.Result{}, err
 		}
+		if req.NamespacedName.Name == servicescm.Name {
+			// Create and retrieve the services ConfigMap as it is not present
+			if configMap, err = r.createServicesConfigMap(ctx, req.NamespacedName); err != nil {
+				return ctrl.Result{}, err
+			}
+		}
 	}
 
-	return ctrl.Result{}, r.reconcileNodes(ctx, configMap)
+	// At this point configMap will be set properly
+	switch req.NamespacedName.Name {
+	case servicescm.Name:
+		// TODO: actually react to changes to the services ConfigMap. Currently do nothing but log
+		r.log.V(1).Info("Reacting to event...", "ConfigMap", req.NamespacedName)
```

Comment:
+1

I suggest `r.log.V(1).Info("Reconciling", "ConfigMap", req.NamespacedName)` and move it up to line [#142](https://github.com/openshift/windows-machine-config-operator/pull/1002/files#diff-f3ec2e06fa75f3747dab6fac751809ce87a96e02263980287e9d9477830049aaR142), so it will also log the _Unexpected_ case.